module/VuFind/src/VuFind/RecordTab/SimilarItemsCarousel.php

```diff
@@ -56,14 +56,25 @@ class SimilarItemsCarousel extends AbstractBase
      */
     protected $searchService;
 
+    /**
+     * Configuration
+     *
+     * @var \Laminas\Config\Config
+     */
+    protected $config;
+
     /**
      * Constructor
      *
-     * @param \VuFindSearch\Service $search Search service
+     * @param \VuFindSearch\Service  $search Search service
+     * @param \Laminas\Config\Config $config Configuration
      */
-    public function __construct(\VuFindSearch\Service $search)
-    {
+    public function __construct(
+        \VuFindSearch\Service $search,
+        \Laminas\Config\Config $config
```

Comment:
Might be worth defaulting this parameter to null to make it optional -- this would improve backward compatibility if anybody is using an outdated factory.

Reply:
Done.